examples/validator/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/instructor-ai/instructor-go/pkg/instructor"
+	openai "github.com/sashabaranov/go-openai"
+)
+
+// User contains user information
+type User struct {
```

Comment:
structs still need `json` and `jsonschema` tags. Curious why this still works without `jsonschema`